internal/grpc/services/storageprovider/storageprovider.go

```diff
@@ -172,6 +172,10 @@ func New(ctx context.Context, m map[string]interface{}) (rgrpc.Service, error) {
 		return nil, err
 	}
 
+	if fs == nil {
```

Comment:
Should `getFS` not return an error if `fs` is `nil`?

Reply:
`getFS` returns `f(ctx, c.Drivers[c.Driver])`, which itself may return `nil`